mobile/src/main/java/ch/epfl/sdp/mobile/ui/profile/ProfileScreenState.kt

```diff
@@ -15,7 +15,7 @@ interface ProfileScreenState : Person {
   val pastGamesCount: Int
 
   /** List of chess matches */
-  val matches: List<ChessMatch>
+  var matches: List<ChessMatch>
```

Comment:
The consumers of `ProfileScreenState` should not be able to set the value of the `matches`, so this should remain a `val`.

```suggestion
  val matches: List<ChessMatch>
```